app/src/main/java/com/asfoundation/wallet/topup/payment/AdyenTopUpFragment.kt

```diff
@@ -358,6 +352,25 @@ class AdyenTopUpFragment : DaggerFragment(), AdyenTopUpView {
     })
   }
 
+  override fun hideBonus() {
+    bonus_layout.visibility = INVISIBLE
+    bonus_msg.visibility = INVISIBLE
```

Comment:
There shouldn't be any case where in the middle of the flow the bonus will change it's visibility so its better to put the view gone instead of visible to reduce space on the view